layouts/_default/material.html

```diff
@@ -8,6 +8,7 @@
   <script src="https://cdn.tailwindcss.com"></script>
 </head>
 <body class="bg-gray-50 min-h-screen text-gray-900">
+  {{ partial "sidebar.html" (dict "context" . "disableSidebar" true) }}
```

Comment:
This is tricky, I still don’t understand why the sidebar isn’t working in custom layouts, even after reviewing the source code :shipit: